src/issueFinder.ts

```diff
@@ -56,11 +58,20 @@ export class IssueFinder {
             createdAt: node.createdAt,
             repositoryNameWithOwner: node.repository.nameWithOwner,
             url: node.url,
+            updatedAt: node.createdAt,
+            labels: this.convertFromGitHubLabels(node.labels.edges)
         };
 
         return issue;
     }
 
+    private convertFromGitHubLabels(edges: Array<{ node: { name: string}}>) {
+        const labels = edges.reduce((labels: string[], edge) => {
```

Comment:
Would this be simpler as a .map that just converts an array of edges to an array of strings?

Reply:
Oh gosh. So much simpler. Updated this to use map